src/runtime/storageNG/handle.ts

```diff
@@ -415,6 +415,10 @@ export class SingletonHandle<T> extends Handle<CRDTSingletonTypeRecord<Reference
     if (!this.canWrite) {
       throw new Error('Handle not writeable');
     }
+    // Sync before clearing (for a similar behaviour to collections).
```

Comment:
I think we could document better eg
"Sync before clearing in order to get an updated versionMap. This ensures we can clear values set by other actors."
(and the same for the kotlin one)

Reply:
Done